demos/projects/ESPRESSIF/adu/main/azure_iot_freertos_esp32_pnp_model.c

```diff
@@ -259,40 +259,49 @@ void vHandleWritableProperties( AzureIoTHubClientPropertiesResponse_t * pxMessag
                 return;
             }
 
-            LogInfo( ( "Verifying JWS Manifest" ) );
-            xAzIoTResult = JWS_ManifestAuthenticate( xAzureIoTAduUpdateRequest.pucUpdateManifest,
-                                                     xAzureIoTAduUpdateRequest.ulUpdateManifestLength,
-                                                     xAzureIoTAduUpdateRequest.pucUpdateManifestSignature,
-                                                     xAzureIoTAduUpdateRequest.ulUpdateManifestSignatureLength,
-                                                     ucADUScratchBuffer,
-                                                     sizeof( ucADUScratchBuffer ) );
-
-            if( xAzIoTResult != eAzureIoTSuccess )
-            {
-                LogError( ( "JWS_ManifestAuthenticate failed: JWS was not validated successfully: result 0x%08x", xAzIoTResult ) );
-                return;
-            }
-
-            xRequestDecision = prvUserDecideShouldStartUpdate( &xAzureIoTAduUpdateRequest );
-
-            xAzIoTResult = AzureIoTADUClient_SendResponse(
-                &xAzureIoTADUClient,
-                &xAzureIoTHubClient,
-                xRequestDecision,
-                ulPropertyVersion,
-                pucWritablePropertyResponseBuffer,
-                ulWritablePropertyResponseBufferSize,
-                NULL );
-
-            if( xAzIoTResult != eAzureIoTSuccess )
+            if( xAzureIoTAduUpdateRequest.xWorkflow.xAction == eAzureIoTADUActionApplyDownload )
```

Comment:
nit: replace the `if` with a `switch` to account for all enum values (current and in the future) as well as a `default` action.